dds_web/api/errors.py

```diff
@@ -236,7 +236,20 @@ def __init__(self, project, message="No key found for current project"):
         general_logger.warning(self.message)
 
 
-####################################################################################################
+class NoSuchInviteError(exceptions.HTTPException):
+    """No invite found in database"""
+
+    def __init__(self, email, message="There is no invitation for the found email adress"):
+        self.message = f"{message}: '{email}'"
+        super().__init__(self.message)
+
+        general_logger.warning(self.message)
+
+
+# ----------------------------------------------------------------------------------- #
+
+
+# ----------------------------------------------------------------------------------- #
 
 
 errors = {
```

Comment:
Do you consider including NoSuchInviteError in the set of errors and assign a response status?

Reply:
Ah, yes. Will add that. `FORBIDDEN` or `INTERNAL_SERVER_ERROR`? Since it should exist but it doesn't so something has gone wrong.

Reply:
@worukan Which code is best here? Forbidden is for when the clients identity is known. `NOT_ACCEPTABLE`? Or something else?

Reply:
Under which circumstance(s) will this be triggered?

Reply:
When a invited user clicks on the emailed link, and for some reason there is now row in the `Invite` table that matches the link.

Reply:
We could use bad request for this. It should not happen but, in case it happens, the request refers to a non-existing entry which is a bad request.